sdk/cosmos/azure-cosmos/azure/cosmos/_service_response_retry_policy.py

```diff
@@ -18,6 +18,7 @@ def __init__(self, connection_policy, global_endpoint_manager, pk_range_wrapper,
         self.pk_range_wrapper = pk_range_wrapper
         self.total_retries = len(self.global_endpoint_manager.location_cache.read_regional_routing_contexts)
         self.failover_retry_count = 0
+        self.max_write_retry_count = 1
```

Comment:
do we also need to recalculate the self.total_retries based on write_regional_routing_context?

Reply:
not sure I understand the question here - truthfully in either situation we would only want a single retry to go out (retryable writes is not meant to be an entire policy)

Reply:
nm, missed the fact that total retries are only use for read operations